test/services/submission_services/run_tests_test.rb

```diff
@@ -12,5 +12,20 @@ class RunTestsTest < ActiveSupport::TestCase
       )
       RunTests.(submission.uuid, submission.solution)
     end
+
+    test "uses language_track" do
+      research_track = create :track, slug: "research_123"
+      ruby_track = create :track, slug: "ruby"
+      exercise = create :exercise, track: research_track, slug: "ruby-1-b"
+
+      submission = create :submission, solution: create(:research_experiment_solution, exercise: exercise)
+
+      RestClient.expects(:post).with('http://test-runner.example.com/submissions',
```

Comment:
Just to be safe: research solutions are put in a different mentoring queue from the regular one, right?

Reply:
It's a different type of solution (`Research::ExperimentSolution`), so they are treated differently with regards to mentoring queues.

For test-running, the only thing using test running right now is the experiments, so there's no overlap.

Reply:
Cool! The PR looks great